virttest/qemu_vm.py

```diff
@@ -1053,8 +1053,12 @@ def add_memorys(devices, params):
                 name = output[0]
                 backend_options = {}
                 backend_options["size_mem"] = "%sM" % params["mem"]
-                if params.get("vm_mem_backend"):
+                mem_backend = params.get("vm_mem_backend")
+                if mem_backend:
                     backend_options["backend_mem"] = params.get("vm_mem_backend")
+                if not params.get("mem-path_mem") and mem_backend == \
+                        "memory-backend-file":
```

Comment:
```suggestion
                if (not params.get("vm_mem_mem-path") and mem_backend ==
                        "memory-backend-file"):
```

And when we have `vm_mem_mem-path`, we need to set `backend_options["mem-path_mem"] = params["vm_mem_mem-path"]` to make it work.

Reply:
Updated